docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_basic_information.yml

```diff
@@ -267,14 +270,14 @@ content: |
   be **from**. If you are requesting the documents to be sent directly to
   you, you will need a release of information.
 ---
-sets:
-  - requestor.name.first
 id: requestor name
 question: |
   Who are you?
 fields:
-  - code: |
-      requestor.name_fields()
+  - First name: requestor.name.first
```

Comment:
Better way to remove suffix is by doing `requestor.name_fields(show_suffix=False)`